client/src/App.tsx

```diff
@@ -1,17 +1,17 @@
 import React from "react";
 import "./App.css";
```

Comment:
are we using this??

Reply:
No, It's not useful as of now. So, I will remove it for now.